tket/src/OpType/OpTypeFunctions.cpp

```diff
@@ -71,6 +71,20 @@ const OpTypeSet& all_multi_qubit_types() {
 }
 
 const OpTypeSet& all_single_qubit_types() {
+  static const OpTypeSet optypes{
+      OpType::noop,        OpType::Z,   OpType::X,  OpType::Y,
+      OpType::S,           OpType::Sdg, OpType::T,  OpType::Tdg,
+      OpType::V,           OpType::Vdg, OpType::SX, OpType::SXdg,
+      OpType::H,           OpType::Rx,  OpType::Ry, OpType::Rz,
+      OpType::PhaseGadget, OpType::U1,  OpType::U2, OpType::U3,
+      OpType::PhasedX,     OpType::TK1};
+  static std::unique_ptr<const OpTypeSet> gates =
+      std::make_unique<const OpTypeSet>(optypes);
+  return *gates;
+}
+
+// the set of OpTypes that implement Gate_ptr->get_tk1_angles()
+const OpTypeSet& all_single_qubit_unitary_types() {
```

Comment:
I think you have these the wrong way round!

Reply:
Yes, I've already fixed this locally. I'm adding a test but got stuck in a weird git manip.

Reply:
Almost there!